icons/phpunit.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.0" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="m 21.46,10.086 c -0.224,0.59 -1.012,0.848 -1.541,0.269 -1.668,-1.925 -4.936,-1.945 -5.845,0.963 -0.694,2.092 1.035,4.28 4.093,4.003 0.671,-0.053 1.145,0.548 0.934,1.21 l -1.521,4.157 c -0.386,1.023 -1.376,1.574 -2.547,1.191 L 3.384,17.631 C 2.335,17.261 1.76,16.118 2.104,15.091 L 6.419,3.298 C 6.884,2.246 7.687,1.77 8.966,2.107 L 20.615,6.353 c 1.051,0.373 1.625,1.515 1.281,2.541 z m -2.223,2.074 c 0,-1.571 -1.713,-2.559 -3.075,-1.773 -1.363,0.786 -1.363,2.76 0,3.546 1.363,0.786 3.075,-0.202 3.075,-1.773 z" fill="#6963b9" /></svg>
```

Comment:
I was wondering if you could use a color of the [Material Color Palette](https://pkief.github.io/material-color-converter/) for this icon. Would be nice if you could adjust this.